src/ol/style/webgl.js

```diff
@@ -1,7 +1,6 @@
 /**
- * Literal style objects differ from standard styles in that they cannot
- * be functions and are made up of simple objects instead of classes.
- * @module ol/style/literal
+ * WebGL style objects slightly differ from standard flat styles for certain properties
+ * @module ol/style/webgl
```

Comment:
Feels like we should work toward having a single place for these types.  I think it would be preferable to have a single set of types and then to allow the implementations to differ (or be incomplete)  rather than having two sets of types.

Reply:
This is definitely what I'm aiming for. Having a separate type for WebGL renderers is helpful to track of where we currently stand and what is the gap between the two style formats, but it's not a situation that we want to keep eternally.

There is still some work to close this gap though, mainly around the concept of rules; I plan on addressing this later on while optimizing the way features are filtered out before rendering in WebGL.